policy/modules/services/courier.te

```diff
@@ -129,20 +132,34 @@ dev_read_rand(courier_pcp_t)
 # POP3/IMAP local policy
 #
 
-allow courier_pop_t self:capability { setgid setuid };
+allow courier_pop_t self:capability { chown dac_read_search fowner setgid setuid };
+dontaudit courier_pop_t self:capability fsetid;
+allow courier_pop_t self:unix_stream_socket connectto;
```

Comment:
Please add `create_stream_socket_perms` to ensure it can create, listen, accept, etc.

Reply:
ok